scripts/benchmark/utils/chain_plumbing.py

```diff
@@ -1,3 +1,6 @@
+import tempfile
```

Comment:
Ordering within each block (stdlib, 3rd party, own) should be alphabetically (See https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md#import-statements) so `pathlib` should move up. Also, for consistency let's make it

```
from pathlib import (
    Path,
)
```